lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -361,6 +361,8 @@ abstract class BaseSlider<
           }
         }
       };
+  @NonNull
+  private final ViewTreeObserver.OnGlobalLayoutListener onGlobalLayoutListener = onScrollChangedListener::onScrollChanged;
```

Comment:
This is sort of confusing. Can we declare a method, say, `updateLableVisibility()`, and do something like:

```
private final ViewTreeObserver.OnScrollChangedListener onScrollChangedListener = this::updateLableVisibility;
private final ViewTreeObserver.OnGlobalLayoutListener onGlobalLayoutListener = this::updateLableVisibility;
```

instead?

Reply:
Thank you very much. I tried. I'm not sure if the name of the method is appropriate.